tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -193,7 +193,7 @@ stages:
     DoCompliance: ${{ parameters.DoCompliance }}
     DoEsrp: ${{ parameters.DoEsrp }}
     stage_name_suffix: gpu
-    EnvSetupScript: setup_env_cuda_11.bat
+    EnvSetupScript: setup_env_cuda.bat
```

Comment:
Please manually run this pipeline to check your change.

Reply:
It failed because issue on MacOS, which is not related to this change.